selfdrive/car/honda/carstate.py

```diff
@@ -433,14 +487,28 @@ def update(self, cp):
       self.cruise_speed_offset = calc_cruise_offset(cp.vl[0x37c]['CRUISE_SPEED_OFFSET'], self.v_ego)
       self.park_brake = 0  # TODO
       self.brake_hold = 0
+    elif self.rdx:
+      can_gear_shifter = cp.vl[0x188]['GEAR_SHIFTER']
+      self.angle_steers = cp.vl[0x156]['STEER_ANGLE']
+      self.angle_steers_rate = cp.vl[0x156]['STEER_ANGLE_RATE']
+      self.gear = cp.vl[0x188]['GEAR']
+      self.cruise_setting = cp.vl[0x1A6]['CRUISE_SETTING']
+      self.cruise_buttons = cp.vl[0x1A6]['CRUISE_BUTTONS']
+      self.main_on = cp.vl[0x1A6]['MAIN_ON']
+      self.blinker_on = cp.vl[0x294]['LEFT_BLINKER'] or cp.vl[0x294]['RIGHT_BLINKER']
+      self.left_blinker_on = cp.vl[0x294]['LEFT_BLINKER']
+      self.right_blinker_on = cp.vl[0x294]['RIGHT_BLINKER']
+      self.cruise_speed_offset = -0.3
+      self.park_brake = 0  # TODO
+      self.brake_hold = 0  # RDX doesn't have EPB  
 
     self.gear_shifter = parse_gear_shifter(can_gear_shifter, self.acura)
```

Comment:
so, rdx gear shifter is parsed as the civic?

Reply:
RDX matches civic, 0x8-D, 0x1-P, 0x2-R, 0x4-N